.github/pull_request_template.md

```diff
@@ -33,14 +33,17 @@ If you need to provide more details, please use the description section.
 -->
 
 - [ ] All files present ( component - md - scss - spec - stories )
-- [ ] Are all props ok and documented
-- [ ] Required props and default values
+- [ ] All props ok and documented (propTypes)
```

Comment:
I think we should also mention typescript types here as we're in the process of moving to typescript which means that propTypes aren't being used for the components which are in typescript.
```suggestion
- [ ] All props ok and documented (propTypes/typescript types)
```